src/components/super-networked-interactable.js

```diff
@@ -93,6 +107,19 @@ AFRAME.registerComponent("super-networked-interactable", {
     }
   },
 
+  _syncCounterRegistration: function() {
+    const el = this.networkedEl;
+    if (!el) return;
+
+    const isPinned = el.components["pinnable"] && el.components["pinnable"].data.pinned;
```

Comment:
```suggestion
    const isPinned = el.getAttribute("pinnable", "pinned");
```

Reply:
this seems to not be a thing, the second arg isn't valid. but single arg works